docs/auth/oauth2proxy/provider.md

```diff
@@ -0,0 +1,119 @@
+---
+id: provider
```

Comment:
Thank you for adding docs as well! For them to show up in the sidebar it needs an addition in https://github.com/backstage/backstage/blob/fde22197c0c845da5d13729be8cffe2ccc6be942/microsite/sidebars.json#L222 too, and then it's nice to add an entry to https://github.com/backstage/backstage/blob/fde22197c0c845da5d13729be8cffe2ccc6be942/mkdocs.yml#L139 too x)

Reply:
Adjusted 👍